experimental/traffic-portal/src/app/api/cdn.service.ts

```diff
@@ -119,4 +119,19 @@ export class CDNService extends APIService {
 
 		return this.put<ResponseCDN>(`cdns/${id}`, body).toPromise();
 	}
+
+	/**
+	 * Queues or dequeues updates on a CDN's servers.
+	 *
+	 * @param cdn The CDN to queue or dequeue updates on.
+	 * @param action The action to perform on the CDN, either "queue" or "dequeue".
+	 */
+	public async queueCDNUpdates(cdn: ResponseCDN | number, action: "queue" | "dequeue"): Promise<CDNQueueResponse> {
```

Comment:
This method appears to be missing from the testing mock service

Reply:
> This method appears to be missing from the testing mock service

Added in d2b9032482